src/java.base/share/classes/java/lang/CharacterData01.java.template

```diff
@@ -25,11 +25,13 @@
 
 package java.lang;
 
+import jdk.internal.vm.annotation.Stable;
+
 /** The CharacterData class encapsulates the large tables once found in
```

Comment:
Taking this opportunity, I'd rectify the comment here (and in other templates too): `CharacterData` -> `CharacterData01`